localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -486,7 +486,14 @@ void NDTScanMatcher::callback_sensor_points(
   }
 
   // covariance estimation
-  std::array<double, 36> ndt_covariance = output_pose_covariance_;
+  Eigen::Quaterniond map_to_base_link_quat = Eigen::Quaterniond(
+    result_pose_msg.orientation.w, result_pose_msg.orientation.x, result_pose_msg.orientation.y,
+    result_pose_msg.orientation.z);
+  Eigen::Matrix3d map_to_base_link_rotation = map_to_base_link_quat.normalized().toRotationMatrix();
```

Comment:
Pleas define as const variable
```suggestion
  const Eigen::Quaterniond map_to_base_link_quat = Eigen::Quaterniond(
    result_pose_msg.orientation.w, result_pose_msg.orientation.x, result_pose_msg.orientation.y,
    result_pose_msg.orientation.z);
  const Eigen::Matrix3d map_to_base_link_rotation = map_to_base_link_quat.normalized().toRotationMatrix();
```